File: go/sumit/main_test.go
package main

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestSumIt(t *testing.T) {
	assert.Equal(t, 12, SumIt(3, 5))
	assert.Equal(t, 22, SumIt(4, 7))
	assert.Equal(t, 42, SumIt(3, 9))
}

File: python/sum_it/main_test.py
from main import sum_it


def test_sum_it():
    assert(sum_it(3, 5) == 12)
    assert(sum_it(4, 7) == 22)
    assert(sum_it(3, 9) == 42)


if __name__ == '__main__':
    test_sum_it()
